benchmarks/README.md

```diff
@@ -19,11 +19,31 @@
 
 # DataFusion Benchmarks
 
-This crate contains benchmarks based on popular public data sets and open source benchmark suites, making it easy to
-run real-world benchmarks to help with performance and scalability testing and for comparing performance with other Arrow
-implementations as well as other query engines.
+This crate contains benchmarks based on popular public data sets and
+open source benchmark suites, making it easy to run more realistic
+benchmarks to help with performance and scalability testing of DataFusion.
 
-## Benchmark derived from TPC-H
+# Benchmarks Against Other Engines
+
+DataFusion is included in the benchmark setups for several popular
+benchmarks that compare performance with other engines. For example:
+
+* [ClickBench] scripts are in the [ClickBench repo](https://github.com/ClickHouse/ClickBench/tree/main/datafusion)
+* [H2o.ai `db-benchmark`] scripts are in [db-benchmark](db-benchmark) directory
+
+[ClickBench]: https://github.com/ClickHouse/ClickBench/tree/main
+[H2o.ai `db-benchmark`]: https://github.com/h2oai/db-benchmark
+
+# Running the benchmarks
+
+## Generating Data
+
+Please use the [bench.sh] script to generate data
+
+
+# Benchmark Descriptions:
+
+## `tpch` Benchmark derived from TPC-H
 
 These benchmarks are derived from the [TPC-H][1] benchmark. And we use this repo as the source of tpch-gen and answers:
```

Comment:
I next hope / plan tor review the other benchmarks and consolidate them and their data generation and runner scripts into the bench.sh framework